kitsune/wiki/views.py

```diff
@@ -1659,7 +1659,8 @@ def what_links_here(request, document_slug):
         return HttpResponseRedirect(url)
 
     links = {}
-    for link_to in doc.links_to():
+    links_to = doc.links_to().prefetch_related("linked_from")
```

Comment:
I think `select_related()` works here and would be a bit more efficient, since it would all be done in the same query?
```suggestion
    links_to = doc.links_to().select_related("linked_from")
```

Reply:
Ah - you are right!